common/app/views/fragments/atoms/mediaEmbed.scala.html

```diff
@@ -16,6 +18,73 @@
     </head>
     <body>
         @views.html.fragments.atoms.media(page.atom, displayCaption = displayCaption)
+ <script>
+        var guardian = {
+            isEmbed: true,
+            isModernBrowser: (
+                'querySelector' in document
+                && 'addEventListener' in window
+                && 'localStorage' in window
+                && 'sessionStorage' in window
+                && 'bind' in Function
+                && (
+                    ('XMLHttpRequest' in window && 'withCredentials' in new XMLHttpRequest())
+                    || 'XDomainRequest' in window
+                )
+            ),
+            config: @Html(templates.js.javaScriptConfig(model.MediaAtomEmbedPage(page.atom)).body),
+            adBlockers: { onDetect: [] }
+
+        };
+        @* Decide the Ophan PV ID here so we can share it with Google Analytics *@
+        guardian.config.ophan = {
+            // This is duplicated from
+            // https://github.com/guardian/ophan/blob/master/tracker-js/assets/coffee/ophan/transmit.coffee
+            // Please do not change this without talking to the Ophan project first.
+            pageViewId: new Date().getTime().toString(36) + 'xxxxxxxxxxxx'.replace(/x/g, function () {
+                return Math.floor(Math.random() * 36).toString(36);
+            })
+        };
+        @* Find the Ophan browser ID as well, for sharing with GA *@
+        @Html(templates.inlineJS.nonBlocking.js.ophanConfig().body)
+
+        var docClass = document.documentElement.className;
+
+        @* Get iframes parent url: http://www.nczonline.net/blog/2013/04/16/getting-the-url-of-an-iframes-parent/ *@
+        var parentUrl = (!!window.parent && window.parent !== window) ? document.referrer : '';
+
+        @* Hack to correctly track external embeds which have been incorrectly embedded on theguardian.com *@
+        var sAccount = (/www\.theguardian\.com/.test(parentUrl)) ? 'guardiangu-network' : 'guardiangu-thirdpartyapps';
+
+        window.s_account = sAccount;
```

Comment:
No idea if anything is actually reading this value, I'm thinking not, maybe a legacy Omniture thing? CC @mkopka